vnw/vnw/pipelines.py

```diff
@@ -29,20 +29,22 @@ def process_item(self, item, spider):
         try:
             kv = {kw: item[kw] for kw in REQUIRED_FIELDS}
         except KeyError as e:
-            logger.error('Drop job: %s %s, missing required key %r',
-                         item.get('name', 'MISSING'),
-                         item.get('url', 'MISSING'),
-                         e)
+            logger.error('Drop job: {} {}, missing required key {}'.format(
```

Comment:
we should keep using old format log. Or at least, don't format before passing to logger methods

https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles